### ISSUE TITLE
Pattern priority suggestion

### ISSUE THREAD

**Issue by ray73864** [raoulvdberge/refinedstorage#2370]
One thing that has always bugged me is the inability to have multiple patterns for the same output.

So i'm currently playing Dungeons, Dragons and Space Shuttles (a 1.12 modpack), I have a Mystical Agriculture farm with lots of different essences and a void ore miner.

Lets say I have a recipe that requires iron, both areas are creating iron in a basic form (iron essence and iron ore), at present, you can only have 1 pattern, it would be nice if you could dump multiple patterns for the same output into the system and it'll choose whichever one will be successful.

So basically, if there is not enough iron essence, it checks the other pattern, if that is good, it creates iron, etc...

**Comment by raoulvdberge** [comment]
You can have multiple patterns. RS chooses the correct pattern.

**Comment by LaysDragon** [comment]
Not actually,I have an  "Dungeons, Dragons and Space Shuttles" server. If I have multi pattern for same output,RS only select the first pattern in the network and ignored the second pattern,told you there is missing item and unable to autocraft recipe,eventhough there is  enough item to craft with second pattern.
